confidant/routes/credentials.py

```diff
@@ -201,8 +202,23 @@ def get_credential(id):
         include_credential_pairs = True
 
         if settings.ENABLE_SAVE_LAST_DECRYPTION_TIME:
-            credential.last_decrypted_date = datetime.now()
+            now = datetime.now()
+            credential.last_decrypted_date = now
             credential.save()
+            # Also try to save the archived credential if ID
+            # corresponds to a 'credential'
+            if credential.data_type == 'credential':
```

Comment:
Let's actually just limit this endpoint to credential.data_type == 'credential' up in line 184

Reply:
that causes potential backwards incompatibilities with existing users. 

It's not a huge deal in this case though